app/.env-dist

```diff
@@ -1,3 +1,4 @@
+PRODUCTION=false
```

Comment:
This is our PRODUCTION variable- in every development site, this should be _false_